### ISSUE TITLE
Serialization does not save glyph dirty state

### ISSUE THREAD

**Issue by adrientetar** [trufont/defcon#1]
cc @graphicore

**Comment by graphicore** [comment]
Should it really do so?

**Comment by adrientetar** [comment]
I think so because if I edit a glyph then undo the edit, the glyph will still be dirty and appear as such in the fontView.

**Comment by adrientetar** [comment]
If dirty=False is serialized then it state "This state is when the glyph wasn't edited", but it also means that the undo stack should be cleared when saving..?

**Comment by graphicore** [comment]
> but it also means that the undo stack should be cleared when saving

Not necessarily, but other `dirty=false` information in the undo stack would need to vanish. So, maybe the undo stack would have to know which revision of it is currently saved on disk.
